src/cloudai/workloads/sleep/standalone_command_gen_strategy.py

```
@@ -33,11 +34,21 @@ def _generate_sleep_command(self) -> str:
         test_cmd = self._generate_sleep_command()
+        full_cmd = self.gen_exec_command(store=False)
         self.test_run.output_path.mkdir(parents=True, exist_ok=True)
         with (self.test_run.output_path / self.TEST_RUN_DUMP_FILE_NAME).open("w", encoding="utf-8") as f:
-            trd = TestRunDetails.from_test_run(self.test_run, test_cmd=test_cmd, full_cmd=test_cmd)
+            trd = TestRunDetails.from_test_run(self.test_run, test_cmd=test_cmd, full_cmd=full_cmd)
             toml.dump(trd.model_dump(), f)
 
-    def gen_exec_command(self) -> str:
-        self.store_test_run()
-        return self._generate_sleep_command()
+    def gen_exec_command(self, store: bool = True) -> str:
```

Ruff (0.15.21)

[warning] Boolean-typed positional argument in function definition

(FBT001)